homeassistant/components/homekit/__init__.py

```diff
@@ -60,6 +62,10 @@
     })
 }, extra=vol.ALLOW_EXTRA)
 
+RESET_ACCESSORY_SERVICE_SCHEMA = vol.Schema({
+    vol.Required(ATTR_ENTITY_ID): cv.comp_entity_ids
```

Comment:
Use `cv.entity_ids` or handle `'all'` keyword instead of an entity_id in the service handler.